x-pack/test/cases_api_integration/security_and_spaces/tests/common/comments/post_comment.ts

```diff
@@ -158,9 +161,43 @@ export default ({ getService }: FtrProviderContext): void => {
           owner: 'securitySolutionFixture',
         });
       });
+
+      describe('files', () => {
+        it('should create a file attachment', async () => {
+          const postedCase = await createCase(supertest, getPostCaseRequest());
+
+          const caseWithAttachments = await createComment({
+            supertest,
+            caseId: postedCase.id,
+            params: getFilesAttachmentReq(),
+          });
+
+          const fileAttachment =
+            caseWithAttachments.comments![0] as CommentRequestExternalReferenceSOType;
+
+          expect(caseWithAttachments.totalComment).to.be(1);
+          expect(fileAttachment.externalReferenceMetadata).to.eql(fileAttachmentMetadata);
+        });
+      });
     });
 
     describe('unhappy path', () => {
+      it('400s when attaching a file with metadata that is missing the file field', async () => {
```

Comment:
Could you please open a new `describe('files', () => {` subsection, move these tests inside, and then include a test where the case already has 100 files and we attempt to `createComment` number 101?

Reply:
I agee. The file should have the same tests as in the `bulk_create_attachments.ts` file.

Reply:
Sounds good, there are a few that probably only make sense in `bulk_create_attachments` but I tried to keep them aligned as possible, let me know if I missed anything.